docs/workflows/genomic_characterization/theiaviral.md

```diff
@@ -49,59 +62,69 @@
-
-??? dna "`read_extraction_rank` optional input parameter"
-    By default, the `read_extraction_rank` parameter is set to "family", which indicates that reads will be extracted if they are classified as the taxonomic family of the input `taxon`, including all descendant taxa of the family. Read classification may not resolve to the rank of the input `taxon`, so these reads may be classified at higher ranks. For example, some *Lyssavirus rabies* (species) reads may only be resolved to *Lyssavirus* (genus), so they would not be extracted if the `read_extraction_rank` is set to "species". Setting the `read_extraction_rank` above the inputted `taxon`'s rank can therefore dramatically increase the number of reads recovered, at the potential cost of including other viruses. This likely is not a problem for scarcely represented lineages, e.g. a sample that is expected to include *Lyssavirus rabies* is unlikely to contain other viruses of the corresponding family, Rhabdoviridae, within the same sample. However, setting a `read_extraction_rank` far beyond the input `taxon` rank can be problematic when multiple representatives of the same viral family are included in similar abundance within the same sample. To further refine the desired `read_extraction_rank`, please review the corresponding classification reports of the respective classification software (kraken2 for Illumina and Metabuli for ONT)
-///
-
-/// html | div[style='clear: both;']
-///
+        
+{{ include_md("common_text/theiaviral_inputs.md", indent=8) }}
+
+    === "TheiaViral_Panel"
+
+        The TheiaViral_Panel workflow accepts Illumina paired-end read data. Read file extensions should be `.fastq` or `.fq`, and can optionally include the `.gz` compression extension. Theiagen recommends compressing files with [gzip](https://www.gnu.org/software/gzip/) to minimize data upload time and storage costs. 
+
+        ???+ dna_blue "`taxon_ids` optional input parameter"
+            **The `taxon_ids` parameter is required for TheiaViral_Panel to run correctly, but is optional in Terra.** 
+            
+            By default, TheiaViral_Panel uses a list of **172** taxon IDs derived from a list of targeted viruses and subtypes in the Viral Surveillance Panel version 2 (VSP v2) produced by Illumina, though this workflow is not specific to that assay. This list can be modified to include or exclude any taxon IDs of interest; however, the taxon IDs _must_ be present in the Kraken2 database used for read classification. Changing this parameter will change what organisms are extracted for assembly and characterization. The list of default taxon IDs can be found below:
```

Comment:
link to Illumina VSP panel list here [a list of targeted viruses and subtypes in the Viral Surveillance Panel version 2 (VSP v2) produced by Illumina\](url-to-list)

also reiterate that the taxon IDs must be found in the kraken database used